frontend/src/app.css

```diff
@@ -1,3 +1,9 @@
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
+
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap');
+
+html {
+	font-family: 'DM Sans', sans-serif;
+}
```

Comment:
not this, we should ship everything locally.
download font files and config on here.